package.json

```diff
@@ -11,13 +11,13 @@
   "scripts": {
     "start": "cross-env NODE_ENV=development rollup -c rollup.config.ts -w",
     "clean": "rimraf lib node/**/*.js",
-    "lint": "eslint '{cli,config,src,test}/**/*.ts'",
+    "lint": "eslint \"{cli,config,src,test}/**/*.ts\"",
     "build": "yarn clean && cross-env NODE_ENV=production rollup -c rollup.config.ts",
     "test": "yarn test:unit && yarn test:integration",
     "test:unit": "cross-env BABEL_ENV=test jest --runInBand",
-    "test:integration": "node --max_old_space_size=8000 node_modules/.bin/jest --config=test/jest.config.js --runInBand",
+    "test:integration": "node --max_old_space_size=8000 node_modules/jest/bin/jest.js --config=test/jest.config.js --runInBand",
     "test:smoke": "config/scripts/smoke.sh",
-    "test:focused": "node_modules/.bin/ts-node --project=test/tsconfig.json test/focusedTest.ts",
+    "test:focused": "node node_modules/ts-node/dist/bin.js --project=test/tsconfig.json test/focusedTest.ts",
```

Comment:
this doesn't seem right does it? should probably be `node -r ts-node/register`

Reply:
I'm on a Windows environment 👀
Do you want me to try it out?

Reply:
> I'm on a Windows environment 👀
> Do you want me to try it out?

That'd be great. I've tested on both and it works, but I think it's supposed to use register? I'm not 100% sure tho. Have to read up further on it

Reply:
Read up a little further on this. Being that this `test:focused` command is designed to only run one `ts` file, this should be fine as is.

You should be able to run the following with no issues on windows (I've tested with `PowerShell` _(only runs from an elevated prompt)_ and `PowerShell 7`):

1. `yarn test:focused test/rest-api/basic.mocks.ts`
2. `yarn test`

Regarding `wsl` usage: that seems to be a can of worms and the instructions for getting puppeteer to work properly would be different depending on the distro someone was using. 😞